src/app_service/service/accounts/dto/create_account_request.nim

```diff
@@ -40,6 +40,7 @@ type
 
     keycardInstanceUID*: string
     keycardPairingDataFile*: string
+    apiConfig*: Option[JsonNode]
```

Comment:
Can we please make it not a `JsonNode`, but a real structure type? e.g. like `ImageCropRectangle` is passed here.

We could leave it as is temporarily, but it shouldn't really take much time to introduce a struct type. And a `defaultApiConfig` method.

Reply:
Sure!